src/__screenshot_tests__/avatar-screenshot-test.tsx

```diff
@@ -0,0 +1,80 @@
+import {openStoryPage, screen} from '../test-utils';
+
+const sizes = [24, 40, 64, 96];
+
+test.each(sizes)('Avatar with numeric badge. Size: %d', async (size) => {
```

Comment:
I'm testing for 4 sizes because there's logic (text size, icon size, badge placement) that is different on different avatar sizes

Will try to group to see if there's any difference

Reply:
I've removed some cases and joined some tests. The time has been reduced from 38s to 25s